platforms/linux-x86_64-arch-gcc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "id": "linux-x86_64-arch-gcc",
+    "name": "RPM",
+    "distribution": "Arch",
+    "release": "2019.04.01",
+    "install-commands": "pacman -Syu && pacman -S --noconfirm ${sysreqs}"
```

Comment:
How about the case for a package from AUR? For example `bwidget`, `jags`, `udunits`, `saga-gis`?

Reply:
Hmm. Yea, that's a good point. There are a lot of useful packages in the AUR that would be worth including, but `pacman -S` obviously will not work with those. If something like a `build-commands` were added, I suppose one could use that to install something like [yay](https://github.com/Jguer/yay) to help with installing packages from the AUR, but I'm not sure if the added complexity is warranted / if that is in the desired scope of sysreqsdb.